components/table/table-col-sort-button.js

```diff
@@ -37,30 +37,39 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 				align-items: center;
 				background-color: transparent;
 				border: none;
+				box-sizing: border-box;
 				color: inherit;
 				cursor: pointer;
 				display: inline-flex;
 				font-family: inherit;
 				font-size: inherit;
+				gap: 12px;
+				height: var(--d2l-sortable-button-height);
 				letter-spacing: inherit;
 				margin: 0;
-				padding: 0;
+				padding: 0.75rem;
 				text-decoration: none;
+				width: var(--d2l-sortable-button-width);
 			}
 			button::-moz-focus-inner {
 				border: 0;
 			}
 			button:disabled {
 				opacity: 0.5;
 			}
-			button:hover {
-				text-decoration: underline;
-			}
 			button:focus-visible,
 			button:${unsafeCSS(getFocusPseudoClass())} {
-				border-radius: 0.2rem;
+				border-radius: calc(var(--d2l-sortable-button-border-radius) * 1.5); /* To get 6px (4px * 1.5) in light mode, will remain 0 in normal case */
```

Comment:
As far as I know these values aren't really dependant on one another and should be separately specified.